### DISCUSSION TITLE
Does anyone know how to change the task queue?

### DISCUSSION THREAD

**Discussion by huynhvuthienloc** [FlareSolverr/FlareSolverr#758] [Q&A]
I am doing the multithread, and the task queue depth is making is slow. My server has a lot of RAM, so I want to change the task queue as high as possible. I am using the Windows version; anyone knows how to change it?
Thank you so much!

**Comment by ngosang** [comment]
We are using Waitress as web server. By default it only accepts 4 concurrent requests. You can edit the following line ti increase the number of threads: `threads = 20`

https://github.com/FlareSolverr/FlareSolverr/blob/cedb7bc54ef0fc1ec21ea6df716e54ce4b9c41c8/src/flaresolverr.py#L111

https://docs.pylonsproject.org/projects/waitress/en/stable/arguments.html
